core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -433,7 +436,11 @@ public void handleClientOperation(
         } catch (SqlException e) {
             reportError(e.getPosition(), e.getFlyweightMessage(), 0);
         } catch (CairoException e) {
-            reportError(-1, e.getFlyweightMessage(), e.getErrno());
+            if (e.isInterruption()) {
```

Comment:
would be good to  not catch query plans with exceptions but it can be another change

Reply:
Marking this one as resolved, but indeed it would be good to do separately.